node/subsystem/src/lib.rs

```diff
@@ -220,4 +227,41 @@ impl<C: SubsystemContext> Subsystem<C> for DummySubsystem {
 			future,
 		}
 	}
-}
\ No newline at end of file
+}
+
+/// This module reexports prometheus types and defines `Metrics` trait.
+pub mod metrics {
```

Comment:
It might the wrong place for this module if we to use it for things other than subsystems and jobs. Not sure what the right place would be.